src/js/controller/model.js

```diff
@@ -256,6 +260,9 @@ define([
                 this.playbackComplete();
             }
 
+            // Restore the playback rate to the provider in case it changed while detached and we reused a video tag.
+            this.setPlaybackRate(this.get('playbackRate'));
```

Comment:
I agree that `model.playbackRate` should reflect the current playback rate of the player, but when the API sets playbackRate, we should set `defaultPlaybackRate` as the desired playbackRate, and then allow the providers and media controller logic in the model do the rest.

See https://developer.mozilla.org/en-US/Apps/Fundamentals/Audio_and_video_delivery/WebAudio_playbackRate_explained